ui/app/components/sidebar/nav/cluster.hbs

```diff
@@ -0,0 +1,86 @@
+<HcAppFrame::Nav as |Nav|>
+  <Nav.Header data-test-sidebar-nav-heading="Vault">Vault</Nav.Header>
+
+  <Nav.Link
+    @route="vault.cluster.secrets"
+    @current-when="vault.cluster.secrets vault.cluster.settings.mount-secret-backend vault.cluster.settings.configure-secret-backend"
+    data-test-sidebar-nav-link="Secrets engines"
+  >
+    Secrets engines
+  </Nav.Link>
+  {{#if (has-permission "access")}}
+    <Nav.Link
+      @route={{get (route-params-for "access") "route"}}
+      @models={{get (route-params-for "access") "models"}}
+      @current-when="vault.cluster.access vault.cluster.settings.auth"
+      class="is-grouped-split"
+      data-test-sidebar-nav-link="Access"
+    >
+      Access
+      <Icon @name="chevron-right" />
+    </Nav.Link>
+  {{/if}}
+  {{#if (has-permission "tools")}}
+    <Nav.Link
+      @route="vault.cluster.tools.tool"
+      @models={{get (route-params-for "tools") "models"}}
```

Comment:
looks like we only do this in one other place, the cluster.hbs. Seems to come from our helpers. Since we haven't used it outside of cluster, are there any concerns about using it here? Or maybe we can add some documentation to the helper itself to clarify what it does.

Reply:
My approach while building these components was to basically leave all the `LinkTo` args as is and the links should still work since `Nav.Link` is just a wrapper component around `LinkTo`. I tested all of the links manually and they were working. For additional context, this component will be dropped in place of the `NavHeader` in the cluster template.